File: BlogWebAPI/BlogWebAPI/Entities/Blog.cs
using System.Text.Json.Serialization;

namespace BlogWebAPI.Entities
{
    public class Blog
    {
        public int Id { get; set; }
        public string Name { get; set; } = null!;
        public int IdUser { get; set; }
        public int IdCategory { get; set; }
        public virtual User? User { get; set; }
        public virtual Category? Category { get; set; }
       
        public virtual ICollection<Article>? Articles { get; set; } = null!;
    }
}

File: angular-blog/src/app/models/blog.ts
import { Article } from "./article";
import { Category } from "./category";
import { User } from "./user";


export class Blog{
    id?:number;
    name?:string;
    idUser?:number;
    idCategory?:number;
    user?: User;
    category?:Category
    articles?: Article[];
}